bundles/ch.elexis.core.ui.contacts/src/ch/elexis/core/ui/contacts/views/Patientenblatt2.java

```diff
@@ -477,6 +476,11 @@ public void reloadContent(Object po, InputData ltf) {
 				fields.add(new InputData(extfield, Patient.FLD_EXTINFO, InputData.Typ.STRING, extfield));
 			}
 		}
+
+		int positionForMobileField = (fields.size() + 1 + 2) / 3;
```

Comment:
Anstelle + 1 + 2 könnte auch + 3 geschrieben werden.

Bzw. wenn die 3 hier überall für Anzahl Spalten steht. Bitte eine Konstante `private static int COLUMNCOUNT = 3;` anlegen, die hier und auch z.B in der Zeile 484 zum Anlegen des `InputPanel` verwendet wird.